### ISSUE TITLE
Question about reproducing results

### ISSUE THREAD

**Issue by smirchan** [amzn/fashion-attribute-disentanglement#3]
Hi,

Thanks for providing pre-trained model weights. It appears from the README that we can run

```
export MODELS_DIR="./models/Shopping100k";
python src/eval.py --dataset_name ${DATASET_NAME} --file_root splits/${DATASET_NAME} --img_root ${DATASET_PATH} --load_pretrained_extractor ${MODELS_DIR}/extractor_best.pkl --load_pretrained_memory ${MODELS_DIR}/memory_best.pkl 
```

to reproduce the top-30 accuracy and NCDG of the paper after downloading the pre-trained weights.

I am instead seeing the following results:

<img width="1508" alt="output" src="https://user-images.githubusercontent.com/2547404/137779546-18f4815e-e428-410f-bcfb-9938c7b53bf5.png">

Do you know what the problem could be?

**Comment by lorisbaz** [comment]
Have you been able to successfully run the experiment? Have you tried using git-lfs?

**Comment by smirchan** [comment]
I was unable to run install `git-lfs` (no sudo access) so I could only use the downloaded checkpoints. I never figured out why the evaluation results I was getting were lower, but I no longer need this experiment, so I'm happy to close the issue. Thanks!
